apps/etterlatte-trygdetid/src/main/kotlin/trygdetid/TrygdetidRoutes.kt

```diff
@@ -172,7 +187,11 @@ private fun TrygdetidGrunnlagDto.toTrygdetidGrunnlag(brukerTokenInfo: BrukerToke
         type = TrygdetidType.valueOf(type),
         bosted = bosted,
         periode = TrygdetidPeriode(periodeFra, periodeTil),
-        kilde = Grunnlagsopplysning.Saksbehandler(brukerTokenInfo.ident(), Tidspunkt.now()),
+        kilde = if (brukerTokenInfo.ident() == Vedtaksloesning.PESYS.name) {
```

Comment:
Note to self: her trur eg vi har potensiale for ein (extension?) funksjon brukerTokenInfo.tilGrunnlagsopplysningskilde() etter kvart

Reply:
God ide men senere